portal-web/test/functional/com/liferay/portalweb/macros/DataEngine.macro

```diff
@@ -1378,6 +1378,50 @@ definition {
 		Click(locator1 = "FormViewBuilder#CREATE_NEW_FIELD_SETS");
 	}
 
+	macro gotoEditFieldset{
+		DataEngine.delete(
+			fieldsetLabel = "${fieldsetName}"
+		);
+		DataEngine.gotoFieldsetItemEllipsis(fieldsetName = "${fieldsetName}");
+		MenuItem.click(menuItem = "Edit");
```

Comment:
Please, remember to insert a line break between actions to increase readability.